### DISCUSSION TITLE
I Am Greatly Confused About How This All Is To Work

### DISCUSSION THREAD

**Discussion by whitedavidp** [VR-25/acc#263] [Q&A]
First, let me say that I am new to ACC and I am using the [ACCA app](https://github.com/MatteCarra/AccA) to configure and run things on my new Moto XT2215 phone. Perhaps I should post at ACCA as well but there is no discussion forum there. So, at least for now, I am sticking with here in hopes someone can explain things to this newbie. Thanks in advance...

What I am trying to so is simply have my phone NOT charge the battery above 80% (unless I over-ride that on a one-time basis), for charging to stop once 80% is achieved, and for the phone to discharge until it reaches 60% - at which time charging should be permitted again.

Here is my ACCA setup:
![Screenshot_20240716-210115](https://github.com/user-attachments/assets/65afea2d-5c09-4ebc-a5d0-de9bef7eb382)
What I did was wait until the phone was charged to 80%. Then I unplugged the phone and ran a battery draining app until it reached 63% and I plugged in the charger once again. Even though the the capacity had not yet reached the "resume point" of 60%, charging began immediately. I was expecting the system to continue to discharge before ACC again enabled charging - at which point it would ultimately reach 80% and the cycle would begin again. Clearly this is NOT what is happening.

So either something is awry or my understanding of how this all works is sub-par. I am betting it is the latter :-)

Can someone educate me here? Thanks

**Comment by cl0ck-byte** [comment]
AccA is dead don't use it

**Reply by whitedavidp** [reply]
That is what I did. Now just using ACC itself. Thanks.
